.travis.yml

```diff
@@ -30,3 +30,4 @@ script:
     - composer test:lint-fixer
     - composer test:static
     - composer test:unit
+    - composer test:infection
```

Comment:
Could you somehow update this to only run on PHP >= 7.2? I think it is okay to ignore PHP 7.1 in this case

Reply:
I downgraded infection to an older version which is compatible with PHP 7.1